dali/operators/python_function/dltensor_function.h

```diff
@@ -45,7 +45,7 @@ std::vector<DLMTensorPtr> CastToDLTensorList(py::list &list, Index exp_size, Ind
   DALI_ENFORCE(list.size() == static_cast<size_t>(exp_size),
       "Function called by DLTensorPythonFunction returned tensor list of wrong size at idx "
       + std::to_string(out_idx) + ". Returned list is of a size "
-      + std::to_string(exp_size) + " and should be of size " + std::to_string(exp_size));
+      + std::to_string(list.size()) + " and should be of size " + std::to_string(exp_size));
```

Comment:
Some rebase sideeffect?

Reply:
No, actual fix!

Reply:
No, actual fix! See original code - it printed the expected value twice.

Reply:
Hasn't been that fixed already?